src/components/frame/RouteLayouts/UserNavigation/UserNavigation.tsx

```diff
@@ -39,6 +39,15 @@ const MSG = defineMessages({
 });
 
 const UserNavigation = () => {
+  /**
+   * TODO: Refactor after #67 is done
+   */
+  const { colonyName } = useParams<{ colonyName: string }>();
+  const { data } = useQuery(gql(getFullColonyByName), {
+    variables: { name: colonyName },
+  });
+  const [colony] = data?.getColonyByName?.items || [];
+
```

Comment:
_PS: I'm not saying this because you introduced this here, I'm just ranting!_

This is such bad component architecture. Colony data should not live in here.

But it's nothing we can do right now. Because this is how the design "is".

Hopefully, when we get to the redesign part of it, we'll be able to properly separate user/colony concerns.